style.css

```diff
@@ -0,0 +1,70 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 62.5%;
+    overflow: hidden;
```

Comment:
Using `overflow: hidden;` on your body is often a sign of masking some issue with your CSS